### ISSUE TITLE
CLI migrations:generate doesn't include database schema name in DROP TABLE script

### ISSUE THREAD

**Issue by josephktcheung** [typeorm/typeorm#1403]
I have following config:
```json
{
  "name": "default",
  "type": "postgres",
  "host": "localhost",
  "port": 5432,
  "username": "username",
  "password": "password",
  "database": "test",
  "schema": "testSchema",
  "entities": ["resources/**/*.entity.ts"],
  "migrations": ["migrations/**/*.ts"],
  "cli": {
    "entitiesDir": "resources",
    "migrationsDir": "migrations"
  }
}
```

And my entity is like this:
```ts
import { Entity, PrimaryGeneratedColumn } from 'typeorm';

@Entity()
export class Test {
  @PrimaryGeneratedColumn() id: number;
}
```

If I use cli to generate a migration, it produces the following:
```ts
import { MigrationInterface, QueryRunner } from 'typeorm';

export class AddTestEntity1515029715105 implements MigrationInterface {
  public async up(queryRunner: QueryRunner): Promise<any> {
    await queryRunner.query(
      `CREATE TABLE "testSchema"."test" ("id" SERIAL NOT NULL, PRIMARY KEY("id"))`
    );
  }

  public async down(queryRunner: QueryRunner): Promise<any> {
    await queryRunner.query(`DROP TABLE "test"`);
  }
}
```

You can see that in the down script the SQL command is `DROP TABLE "test"`, but instead it should be `DROP TABLE "testSchema"."test"`, otherwise it will fail when the down script is run: `QueryFailedError: table "test" does not exist`.

**Comment by pleerock** [comment]
Migration generation works only in mysql currently. We are working on complete support for all drivers and planning to release fixes in 0.2.0

**Comment by AlexMesser** [comment]
fixed and released in `typeorm@0.2.0-alpha.23`. You can try it via `npm i typeorm@next`.
